sympy/integrals/risch.py

```diff
@@ -895,7 +896,7 @@ def derivation(p, DE, coefficientD=False, basic=False):
         if basic:
             r += d.as_expr()*pv.diff(v)
         else:
-            r += (d*pv.diff(v)).as_poly(t)
+            r += (d*pv.diff(v)).as_poly(t, field=p.domain.is_Field)
```

Comment:
This should probably be `field=True`.

Reply:
Okk @jksuom . I had commited it in my new commit.

Sorry for the late response, my exams were going previous week.